__tests__/app.test.js

```diff
@@ -124,10 +129,64 @@ describe('api/articles', () => {
             return request(app).get('/api/articles')
                 .expect(200)
                 .then(({ body: { articles }}) => {
+                    expect(articles).toHaveLength(articleData.length)
                     articles.forEach(article => {
                         expect(article).not.toHaveProperty('body')
                     })
                 })
         })
     })
+})
+
+describe('api/articles/:article_id/comments', () => {
+    describe('GET - 200', () => {
+        it('returns all comments for an article, with expected properties', () => {
+            const exampleArticleId = 1
+            const expectedNumberOfComments = 11
+            return request(app).get('/api/articles/' + exampleArticleId + '/comments')
```

Comment:
Whilst I will always shout the praises pf well named variables - at times your code is at risk of becoming less readable because of them. 
`return request(app).get('/api/articles/' + exampleArticleId + '/comments')`
`return request(app).get('/api/articles/1/comments')` is readable enough in my opinion